### PR TITLE
Views

### PR DESCRIPTION
**Pull request by vvoo21**
#### In this milestone, I performed the following requirements:

- Implemented the design from the sneak peek wireframes.
- I was focused on making sure everything functions first.
- Used methods that I had created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+      <li>
+        <%= link_to user_path(user) do %>
+          <div>
+            <%= render "shared/user", user: user %>
+          </div>
+        <% end %>
+      </li>
+    <% end %>
+  </ul>
+</section>
```
Comment (zieeco) [inline app/views/users/index.html.erb]: - Hey, your app works fine, however, you've not followed the give wireframe and your layout design looks totally different. please, improve this by following the given wireframe in implementing your design:


*user index and show page wireframe*
image

*Your user index page*
image

*Your user show page*
image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
+      <%= link_to user_post_path(post.author, post), class: 'user-posts' do %>
+        <%= render "shared/post_info", post: post %>
+        <%= render "shared/post_comments", comments: post.recent_comments %>
+      <% end %>
     <% end %>
-  <% end %>
-</ul>
+  </div>
+  <a class="btn secundary-text" href="#">Pagination</a>
+</section>
```
Comment (zieeco) [inline app/views/posts/index.html.erb]: - The same issue is applicable here:

*User post index and User post show page wireframe*
image

*Your user post index*
image

*your user post show page*

image


**Please, improve on all of these**

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @vvoo21,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice adding the app views. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions


- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @vvoo21 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**